sentry_link/pubspec.yaml

```
@@ -12,7 +12,7 @@ dependencies:
   gql_exec: ">=0.4.0 < 2.0.0"
   gql_link: ">=0.5.0 < 2.0.0"
   gql: ">=0.14.0 < 2.0.0"
-  sentry: ^7.8.0
+  sentry: ">=8.0.0 <9.0.0"
```

Using the `^` is identical to your solution

```suggestion
  sentry: ^8.0.0
```